### DISCUSSION TITLE
Problems when merging multiple scATAC-seq samples

### DISCUSSION THREAD

**Discussion by sjasws** [stuart-lab/signac#1600] [Q&A] [answered]
My multiple scATAC-seq samples were sequenced separately, so I am now going to merge all samples together and then do quality control (relying on nCount, TSS.enrichment score, etc.). However, after I merged the data followed by the vignette (the reduce function used to create the common peak set), I found that compared to the data analyzed alone, indicators such as nCount_peak, nFeature_peak, and TSS.enrichment score have changed dramatically. These metrics were significantly smaller for each cell after merge than when analyzed separately. Is that reasonable? Should I merge these samples before quality control?

**Comment by timoast** [comment]
This should not be the case, if you can open an issue with a reproducible example I can take a look

**Reply by sjasws** [reply, marked as answer]
I'm sorry. I made a mistake. The TSS.enrichment score did not change after merging samples. But I found that nCount_peak and nFeature_peak changed. After I read the concept of these indexs from https://github.com/stuart-lab/signac/discussions/1492, I think it is reasonable because the short peaks would merge as a long peak by constructing a commen peak set using reduce() function, thus the total peak counts in a cell may be reduce. I don't know if I am right?

**Reply by timoast** [reply]
Yes, if you change the set of peaks then nCount_peak and nFeature_peak would change
